Util.js

```diff
@@ -1,13 +1,19 @@
 /* eslint no-unused-vars: ["error", { "varsIgnorePattern": "_" }] */
 /* globals Utilities */
 
+// RegEx test for root path references
+const regexPath_ = /^projects\/.+?\/databases\/\(default\)\/documents\/.+\/.+$/
+// RegEx test for testing for binary data by checking for non-printable characters.
+// Parsing strings for binary data is completely dependent on the data being sent over.
+const regexBinary_ = /[\x00-\x08\x0E-\x1F]/ // eslint-disable-line no-control-regex
```

Comment:
For `const`s at the top level in Google Apps Scripts, I always get a "redeclaration" error. When I try to run some tests that involve using this regex pattern, I get this error: `TypeError: redeclaration of const regexBinary_.`. In the past I've solved these by just making top-level properties `var` instead of `const`. 

Do you get this behavior too? If so, could you change these to `var`?

Other than that, this looks good to go! I'll merge it in once we get this issue resolved.

Reply:
Oh right! Good catch on that! :+1: